server/platform/shared/filestore/filesstore.go

```diff
@@ -125,10 +125,24 @@ func (settings *FileBackendSettings) CheckMandatoryS3Fields() error {
 	return nil
 }
 
+// NewFileBackend creates a new file backend
 func NewFileBackend(settings FileBackendSettings) (FileBackend, error) {
+	return newFileBackend(settings, true)
+}
+
+// NewExportFileBackend creates a new file backend for exports, that will not attempt to use bifrost.
+func NewExportFileBackend(settings FileBackendSettings) (FileBackend, error) {
+	return newFileBackend(settings, false)
+}
```

Comment:
This abstraction seems slightly wrong. At this level, a file backend can either be local or aws s3. But we are also mentioning whether to use bifrost or not which seems like a leak in abstraction.

I'd simply rather call `filestore.NewS3FileBackendWithoutBifrost` from the platform service itself, and avoid the 2 layers of conditions.

Reply:
While this is fitting our use case, nothing would prevent a self-hosted from choosing a "local" driver for their export. I'm afraid that moving the store creation one level higher will just result on code duplication. Unless I'm missing what you mean there?!